src/Jackett.Common/Definitions/rockbox.yml

```diff
@@ -38,7 +38,7 @@
     inputs:
       search: "{{if .Query.Artist}}{{ .Query.Artist }}{{else}}{{ .Keywords }}{{end}}"
     rows:
-      selector: body:nth-child(2) tbody tr td:nth-child(2) table.lista tbody tr:nth-child(2) table.lista tbody tr
+      selector: body:nth-child(2) tbody tr td:nth-child(2) table.lista tbody tr:nth-child(2) table.lista tbody trtr:has(a[href^="download.php?id="])
```

Comment:
I suspect that **trtr** is a typo.
but now that you are filtering you can just use
```yaml
      selector: tr:has(a[href^="download.php?id="])
```